nomenclature/validation_schemas/variable_schema.yaml

```diff
@@ -17,11 +17,15 @@ definitions:
         # The lower-level dictionary are the attributes
         properties:
           unit:
-            type: [ string, "null" ]
+            oneOf:
+              - type: [ string, "null" ]
+              - type: array
+                items:
+                  type: string
         required:
           - unit
         additionalProperties:
```

Comment:
Also just tested this without the `additionalProperties` tag and all unit tests still run through perfectly. Maybe something's not right here.

Reply:
As above, I didn't add a test for all possible cases, because we should have confidence that **jsonschema** works as expected with correct specifications.